.github/workflows/unit-test.yml

```diff
@@ -25,10 +25,9 @@ jobs:
     - name: Install Poetry
       run: |
         curl -sSL https://install.python-poetry.org | python3 -
-    - name: Install dependencies
+    - name: Install dependencies needed to run linting
       run: | 
-        poetry install
-        poetry add flake8 pytest
+        poetry install --only linting
```

Comment:
We suddenly started getting an error about `R_HOME` being missing when installing `rpy2` - I think installing that package depends on R already being installed in the environment. I'm not sure why this suddenly started failing when it didn't used to (fails on `dev` as well), but there's no reason to install all the dependencies here - we just need to run linting. The full dependency list is installed in the Docker image that runs unit tests later in this action.